web/services/hosts_next.go

```diff
@@ -1,17 +1,26 @@
 package services
 
 import (
+	"time"
+
 	"github.com/lib/pq"
+	"github.com/trento-project/trento/internal"
 	"github.com/trento-project/trento/web/entities"
 	"github.com/trento-project/trento/web/models"
 	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
 )
 
+const HeartbeatTreshold = 5 * 2 * time.Second
+
+var timeSince = time.Since
+
 //go:generate mockery --name=HostsNextService --inpackage --filename=hosts_next_mock.go
 type HostsNextService interface {
 	GetAll(filters map[string][]string) (models.HostList, error)
 	GetAllSIDs() ([]string, error)
 	GetAllTags() ([]string, error)
+	Heartbeat(agentID string) error
```

Comment:
I think it's too soon to deal with this type of abstractions. I don't see an immediate value on our code of having an heartbeater interface that from now to an indefinite time would be only fulfilled by one service.

Reply:
I understand what you say. Not going to push on this.
Go ahead like this.

Reply:
Alright!. I think that we will ad interfaces like this pretty soon when we will start to generalize (e.g. TaggableService could be one of them). I just don't see this specific interface as a general behavior in our application, the fact we are using an interface here is more because we need to be able to mock this service in our tests.